crates/orchestrator/src/workers/update_state.rs

```diff
@@ -45,3 +51,10 @@ impl Worker for UpdateStateWorker {
         }
     }
 }
+
+impl UpdateStateWorker {
+    /// To parse the block numbers from the vector of jobs.
+    fn parse_job_items_into_block_number_list(job_items: Vec<JobItem>) -> String {
```

Comment:
Do you have some tests for this new method already?

Reply:
Would be great to cover it already

Reply:
Sure I will add a simple test in this file itself.

Reply:
Added a simple test case